elastic/kibana
# [Fleet] Fix preconfiguration output deletion update managed policies

## Summary

Resolve https://github.com/elastic/kibana/issues/193407

That PR allow to remove an output that was used in a managed agent policies, it was previously failing due to missing force flag, that we use for all operation from the preconfiguration.

## How to test

Try to add that config to your kibana.dev.yml and then remove it should work without issues
```
xpack.fleet.outputs:
  - id: test
    name: test output
    type: elasticsearch
    hosts: ['http://es.fr:9200']
    is_default: false
    is_default_monitoring: false
    config:
      test: 123
xpack.fleet.agentPolicies:
  # Test policy
  - name: Elastic Cloud agent policy
    description: Default agent policy for agents hosted on Elastic Cloud
    id: test-policy
    is_default: false
    data_output_id: test
    is_managed: true
    overrides:
      agent.monitoring:
        apm:
          hosts:
            - <apm host url>
          environment: <apm environment>
          secret_token: <secret token>
    is_default_fleet_server: false
    namespace: default
    monitoring_enabled: ['traces']
    unenroll_timeout: 86400 # 1 day TTL
    package_policies: []
```
